vendor/github.com/tidepool-org/go-common/clients/auth.go

```diff
@@ -235,7 +235,7 @@ func (client *AuthClient) DeleteRestrictedToken(tokenID string, token string) er
 	defer res.Body.Close()
 
 	switch res.StatusCode {
-	case http.StatusNoContent:
```

Comment:
Nit: would it make sense to futureproof this to accept either OK or No Content? Effectively same intent & outcome.